places-solution-integrations/campaign/postbacks.md

```diff
@@ -1,76 +1,86 @@
+{% endhint %}
+
+In this example, a Condition is created that causes the Rule to trigger only for users that are in the city named **San Jose**.
 
-In this example, we will add a condition that will cause our rule to trigger only for users that are in the city named "San Jose".
+1. Under the **Conditions** section, click **Add**.
 
-Click the "Add" button under the "Conditions" section:
+2. From the **Extension** drop-down list, select **Places**.
 
-<img src="../../.gitbook/assets/integrations/campaign/pb-addCondition.png" width="800" /><br />
+3. From the **Condition Type** drop-down list, select **City**.
 
-Select `Places` from the "Extension" dropdown, and choose `City` from the "Condition Type" dropdown. In the UI on the right, type "San Jose" and hit enter. When you're done, hit the "Keep Changes" button.
+4. On the right pane, type **San Jose** and hit **Enter**.
+
+5. Click **Keep Changes**.
 
 <img src="../../.gitbook/assets/integrations/campaign/pb-setCondition.png" width="800" /><br />
 
 ## Define the Action
 
-The last thing we need to do is define the `Action`. First, click the "Add" button under the "Actions" section:
+1. Under the **Actions** section, click **Add**.
 
-<img src="../../.gitbook/assets/integrations/campaign/pb-addAction.png" width="800" /><br />
+2. From the **Extension** drop-down list, select **Mobile Core**.  
 
-Select `Mobile Core` from the "Extension" dropdown and choose `Send Postback` from the "Action Type" dropdown.
+3. From the **Action Type** drop-down list, select **Send Postback**.
 
-We have to first provide the endpoint for our postback. In the URL field, enter the path to the correct table that will hold your location data in campaign. Constructing the correct URL is made easier by using data elements for your ACS Server and PKEY values.
+4. To provide the endpoint for our postback, in the **URL** field, enter the path to the correct table that will hold your location data in Campaign.
 
-Next, check the "Add Post Body" checkbox.
+5. Select the **Add Post Body** checkbox.
 
-In the "Post Body" text area, add in the appropriate values that ACS is expecting in this table. The example below shows how you can use data elements to dynamically populate the `locationData` with correct values for this user.
+6. In the **Post Body** field, add the appropriate values that Campaign Standard is expecting in this table.
 
-<img src="../../.gitbook/assets/integrations/campaign/pb-setAction.png" width="800" /><br />
+7. In the **Content Type**, type **application/json**.
 
-Finally, set the "Content Type" to `application/json` and change the "Timeout" to a non-zero number - we recommend `2`.
+8. In **Timeout**, select a non-zero number, for example, **2**.
 
-Once you're finished, hit the "Keep Changes" button.
+9. Click **Keep Changes**.
 
-## Save the Rule and Rebuild your Property
+<img src="../../.gitbook/assets/integrations/campaign/pb-setAction.png" width="800" /><br />
```

Comment:
Please remove this graphic.

Reply:
there are 4 main screens in each of these topics.  i've left one graphic for each screen after all fields have been entered.  is that reasonable?

Reply:
Yep, that's perfect! thanks @sbenedicadb!